core/src/executor/evaluate/error.rs

```diff
@@ -74,6 +74,9 @@ pub enum EvaluateError {
     #[error("subquery returns more than one row")]
     MoreThanOneRowReturned,
 
+    #[error("subquery recieved more than one column")]
+    MoreThanOneColumnReceived,
```

Comment:
Looks like MoreThanOneColumnReturned sounds more easy to understand.

Reply:
yeah, i totally agree with you,
I have made changes as you guided. Thank you